src-tauri/src/editor/mod.rs

```diff
@@ -62,6 +64,40 @@ pub fn init_editor(app: &AppHandle, video_file: String, size: (u32, u32)) {
     editor_win.set_focus().unwrap();
 }
 
+pub fn show_editor(app: &AppHandle, video_file: String, size: (u32, u32)) {
+    let (width, height) = size;
+
+    let editor_url = WebviewUrl::App(
+        (format!(
+            "/editor?file={}&width={}&height={}",
+            video_file, width, height
+        ))
+        .into(),
+    );
+    let editor_url = Url::parse(&editor_url.to_string()).expect("Invalid URL");
```

Comment:
Is L77 mandatorily needed? Trying to avoid the `Url` crate if we can make do without it.

Reply:
We can opt to remove the `show_editor` function for now, to avoid `Url` crate as it doesn't really enhance the user experience.

Reply:
works